src/main/java/skeltal/Ui.java

```diff
@@ -0,0 +1,39 @@
+package skeltal;
+
+/**
+ * A class that handles interaction with the user, Eg printing responses.
+ */
+public class Ui {
+
+    private static String line = "---------------------------------------------";
+
+    /**
+     * Prints an introduction statement.
+     */
+    public static void introduction() {
```

Comment:
Perhaps this should be renamed to `introduce()`? According to the [guidelines on naming](https://nus-cs2103-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#use-nouns-for-things-and-verbs-for-actions)